lix/packages/sdk/src/discussion/comment.ts

```diff
@@ -0,0 +1,32 @@
+import { Kysely } from "kysely";
+import type { LixDatabaseSchema } from "../database/schema.js";
+import { v4 } from "uuid";
+
+export async function comment(args: {
+	db: Kysely<LixDatabaseSchema>;
+	currentAuthor: string;
+	parentComment: { id: string };
+	body: string;
+}) {
+	return args.db.transaction().execute(async (trx) => {
```

Comment:
Found myself wondering what comment is/does. Using `addComment` would clarify that a comment is added to a discussion.

```suggestion
export async function addComment(args: {
	db: Kysely<LixDatabaseSchema>;
	currentAuthor: string;
	parentComment: { id: string };
	body: string;
}) {
	return args.db.transaction().execute(async (trx) => {
```